csrankings-s.csv

```diff
@@ -1795,7 +1795,7 @@ Sujit Gujar,IIIT Hyderabad,https://www.iiit.ac.in/people/faculty/sujit.g,9yxKajs
 Sujit Kumar Chakrabarti,IIIT Bangalore,http://www.iiitb.ac.in/faculty_page.php?name=SujitKumarChakrabarti,NOSCHOLARPAGE
 Sujoy Ghose,IIT Kharagpur,http://www.myopencourses.com/profile/prof-sujoy-ghosh,swQD_vgAAAAJ
 Sujoy Sikdar,Binghamton University,https://www.binghamton.edu/computer-science/contact/profile.html?id=ssikdar,OmZaKTEAAAAJ
-Sujoy Sinha Roy,University of Birmingham,https://www.birmingham.ac.uk/staff/profiles/computer-science/roy-sinha-sujoy.aspx,58_wa_4AAAAJ
+Sujoy Sinha Roy,Graz University of Technology,https://www.iaik.tugraz.at/person/sujoy-sinha-roy/,58_wa_4AAAAJ

```

Comment:
This institute does not offer PhDs. Please address the inclusion eligibility question as well, since not clearly in CS.

Reply:
Of course does Sujoy offer PhDs at this institute. I offer PhDs at this institute as well.

Reply:
The web page does not list a PhD offering, AFAICT?